### ISSUE TITLE
iOS file list top bar has no clear description

### ISSUE THREAD

**Issue by wuenschedesign** [owncloud/ios-app#212]
### Steps to reproduce
1. Start ownCloud App
2. Use one ownCloud Account
3. Watch File list

### Expected behaviour
Top bar should always tell the user where in the app he is located, e.g.: »File list« or »my files«

### Additional Comments
The right wording should be tested with users or at least discussed in a group to avoid misunderstanding for the user.

### Actual behaviour
Top bar just shows a slash »/«. Technically it means the root directory but a user that is not IT savvy is left in the unknown, what he can expect here. 

### Client
**iOS version:**
12.0.1

**ownCloud app version:**
1.0 build 85

**Device model:**
iPhone 7

### Screenshot
![bildschirmfoto-2019-01-14-um-17 47 38-b](https://user-images.githubusercontent.com/40062503/51138955-de6b6880-1842-11e9-9bc1-535cc81f09ff.png)

**Comment by jesmrec** [comment]
Latest builds (f.ex., build 95 `6f7c4dd`) shows on the top of the view the bookmark name instead the slash in case of root folder, and the name of the folder in case of non-root folder.
